docs/open-pixee.md

```diff
@@ -21,15 +21,12 @@ Learn more in the [README](https://github.com/pixee/pixee-cli#readme)
 Do you want to see Pixee at work in your repository? We've integrated the PixeeCLI into CI scripts for popular platforms.
 
 - [Bitbucket](https://github.com/pixee/pixee-cli/blob/main/ci/bitbucket/README.md)
-
-Coming soon!
-
-- Gitlab
-- Github
+- [GitLab](https://github.com/pixee/pixee-cli/blob/main/ci/gitlab/README.md)
+- [GitHub](https://github.com/pixee/ci-integrations/tree/main/docs/github.md)
```

Comment:
404

Reply:
These were waiting on their own PRs, and should be working now.